### PR TITLE
yt-dlp: update page

### PR DESCRIPTION
**Pull request by thrila**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by thrila** [comment]
@kbdharun please confirm

File: pages/common/yt-dlp.md
Comment (CleanMachine1) [inline pages/common/yt-dlp.md:11]: removes some clarity, unnecessary.

File: pages/common/yt-dlp.md
Comment (CleanMachine1) [inline pages/common/yt-dlp.md:15]: This has removed key information to the user, why?

File: pages/common/yt-dlp.md
Comment (CleanMachine1) [inline pages/common/yt-dlp.md:29]: this isn't the best of changes, how is the user to know that a playlist is needed

File: pages/common/yt-dlp.md
Comment (CleanMachine1) [inline pages/common/yt-dlp.md:35]: This actually needs changing, its windows specific for a `common` page

**Review by CleanMachine1 — CHANGES_REQUESTED** [review]

**Comment by thrila** [comment]
Thanks for the correction

File: pages/common/yt-dlp.md
Comment (fazlearefin) [inline pages/common/yt-dlp.md:37]:
```suggestion
`yt-dlp -P "{{path/to/directory}}" -o "{{%(series)s/%(season_number)s - %(season)s/%(episode_number)s - %(episode)s.%(ext)s}}" "{{url}}"`
```

Please see https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#paths

**Review by Magrid0 — DISMISSED** [review]
Looks good to me

File: pages/common/yt-dlp.md
Comment (kbdharun) [inline pages/common/yt-dlp.md:25]: Thanks for your contribution. Can you update the same in other examples?

Comment (thrila) [reply pages/common/yt-dlp.md:25]: What other examples?

Comment (kbdharun) [reply pages/common/yt-dlp.md:25]: > What other examples?

The placeholder examples above.

**Comment by CleanMachine1** [comment]
You did not make any changes following my comments, and then proceeded to close them, saying they have been resolved.

Please actually respond to them appropriately.

**Review by CleanMachine1 — DISMISSED** [review]
Please do not approve before actually reading and commenting on my original feedback.

File: pages/common/yt-dlp.md
Comment (CleanMachine1) [reply pages/common/yt-dlp.md:37]: FYI, this was marked as resolved without any commits being added.
Do not close comments from maintainers without discussing or doing the suggested change.

**Comment by CleanMachine1** [comment]
Do you plan on changing anything suggested, or should we close this PR?

**Comment by thrila** [comment]
> Do you plan on changing anything suggested, or should we close this PR?

yeah lemme do it now kinda forgot

**Comment by thrila** [comment]
> Do you plan on changing anything suggested, or should we close this PR?

Hi finished do you mind checking?

**Comment by CleanMachine1** [comment]
That commit has nothing in it, did you not push your changes?

File: pages/common/yt-dlp.md
Comment (vitorhcl) [reply pages/common/yt-dlp.md:30]: I agree.

@thrila we try to be as clear as possible, and we use descriptive placeholder. I don't see any benefits on removing the example from the placeholder.

What about reverting this change?

File: pages/common/yt-dlp.md
Comment (vitorhcl) [inline pages/common/yt-dlp.md:16]:
```suggestion
- Download a video or playlist using best MP4 video available (default is "bv\*+ba/b"):
```
What about this? I agree with @CleanMachine1, your change removes key information for the user.

**Review by vitorhcl — DISMISSED** [review]

**Comment by thrila** [comment]
Any more corrections?

**Comment by CleanMachine1** [comment]
I think you are making a mistake with adding the changes, could you share what commands you are running/what you are doing to make the suggested changes.

**Comment by vitorhcl** [comment]
I think he is just merging the branch instead of commit the changes.

@thrila if you are on GitHub Mobile, you meed to click on the comments and then click om the "Commit" button.

**Comment by thrila** [comment]
I am on mobile thanks for the tip 
Still there's one thing that confuses me 
As you can see from the last stream of commits the placeholder was removed and a example URL was placed I don't know if this is something he did for all of them how do I check? Other than that I actually started with an example URL and some dudes said a place holder was better when I made a PR

**Comment by vitorhcl** [comment]
> Still there's one thing that confuses me 
> As you can see from the last stream of commits the placeholder was removed and a example URL was placed

I saw the last commits and it looks like it's the opposite, you replaced the example URLs with the generic placeholder text "url".

> I don't know if this is something he did for all of them how do I check?

Well, the current state is replacing some examples URLs with the "url" placeholder.

Anyways, you can see the changes that would be applied if thisnPR was merged clicking on the first bar below the PR description that has green and red numbers (the quantity of modified lines).

Green = addition, red = removal.

> Other than that I actually started with an example URL and some dudes said a place holder was better when I made a PR

Are you talking about https://github.com/tldr-pages/tldr/pull/12580#discussion_r1545423028? I don't know if @kbdharun thinks it is a good idea or if that was just a suggestion for improving the page consistency.

In my opinion, in this case, example URLs are more user-friendly.

**Comment by kbdharun** [comment]
Are you talking about https://github.com/tldr-pages/tldr/pull/12580#discussion_r1545423028? I don't know if @kbdharun thinks it is a good idea or if that was just a suggestion for improving the page consistency.

Agreed, having example URLs seems much better. I wasn't clear in my previous review comment,  what I meant is since one example was updated to the URL I suggested updating it throughout the page or reverting the link in the specific line (in this case, reverting it seems like the best course of action).

**Comment by thrila** [comment]
> Hi @thrila, we are sending you a friendly nudge this PR still has unresolved comments. Could you please make the suggested changes?

yeah  will do it right now

**Comment by thrila** [comment]
done !! please check to make sure it's the format we want

File: pages/common/yt-dlp.md
Comment (kbdharun) [inline pages/common/yt-dlp.md:34]: This link for the example should be an Udemy one

```suggestion
`yt-dlp -u {{user}} -p {{password}} -P "{{path/to/directory}}" -o "{{%(playlist)s/%(chapter_number)s - %(chapter)s/%(title)s.%(ext)s}}" "{{https://www.udemy.com/java-tutorial}}"`
```

File: pages/common/yt-dlp.md
Comment (kbdharun) [inline pages/common/yt-dlp.md:38]:
```suggestion
`yt-dlp -P "{{path/to/directory}}" -o "{{%(series)s/%(season_number)s - %(season)s/%(episode_number)s - %(episode)s.%(ext)s}}" "{{https://www.youtube.com/watch?v=oHg5SJYRHA0}}"`
```
For consistency with other example links.

File: pages/common/yt-dlp.md
Comment (kbdharun) [reply pages/common/yt-dlp.md:12]:
```suggestion
- List the available downloadable formats for a video:
```

File: pages/common/yt-dlp.md
Comment (kbdharun) [inline pages/common/yt-dlp.md:36]:
```suggestion
- Download the entire series season keeping each series and each season in a separate directory:
```

File: pages/common/yt-dlp.md
Comment (kbdharun) [inline pages/common/yt-dlp.md:32]:
```suggestion
- Download a Udemy course keeping each chapter in a separate directory:
```

File: pages/common/yt-dlp.md
Comment (kbdharun) [inline pages/common/yt-dlp.md:28]:
```suggestion
- Download all playlists of a YouTube channel/user keeping each playlist in a separate directory:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page. After this the PR is GTG.

**Comment by thrila** [comment]
> Thanks for your contribution. I have some suggestions for this page. After this the PR is GTG.

Well ok I'll make the changes

**Comment by thrila** [comment]
can some check to connfirm that the changes were indeed applied?

File: pages/common/yt-dlp.md
Comment (kbdharun) [inline pages/common/yt-dlp.md:30]:
```suggestion
`yt-dlp -o "{{%(uploader)s/%(playlist)s/%(playlist_index)s - %(title)s.%(ext)s}}" "{{https://www.youtube.com/user/TheLinuxFoundation/playlists}}"`
```

File: pages/common/yt-dlp.md
Comment (kbdharun) [inline pages/common/yt-dlp.md:38]:
```suggestion
`yt-dlp -P "{{path/to/directory}}" -o "{{%(series)s/%(season_number)s - %(season)s/%(episode_number)s - %(episode)s.%(ext)s}}" "{{https://videomore.ru/kino_v_detalayah/5_sezon/367617}}"`
```

**Review by kbdharun — APPROVED** [review]
LGTM, as the other maintainers say these links must be updated to align with their example description.

File: pages/common/yt-dlp.md
Comment (spageektti) [reply pages/common/yt-dlp.md:38]: Is this URL valid because I got 403?

Comment (kbdharun) [reply pages/common/yt-dlp.md:38]: > Is this URL valid because I got 403?

Oh, I didn't check for it (just reverted the change), do you have any suggestions for a new one?

Comment (spageektti) [reply pages/common/yt-dlp.md:38]: > > Is this URL valid because I got 403?
> 
> Oh, I didn't check for it (just reverted the change), do you have any suggestions for a new one?

No, because I don't understand this example fully.

Comment (spageektti) [reply pages/common/yt-dlp.md:38]: > Maybe a playlist like [youtube.com/playlist?list=PLqE7PUanPIFTs1Zy1Lz270nUHhtZcx66e&si=VTWFdggY2cbLCLsP](https://youtube.com/playlist?list=PLqE7PUanPIFTs1Zy1Lz270nUHhtZcx66e&si=VTWFdggY2cbLCLsP)?

Using the command with this playlist resulted in a folder like this:
```
└── NA
    └── NA - NA
        ├── NA - NA.mkv
        └── NA - NA.webm
```
Maybe just remove the example?

**Comment by spageektti** [comment]
@thrila can you please check the suggestions?

File: pages/common/yt-dlp.md
Comment (spageektti) [reply pages/common/yt-dlp.md:38]: @kbdharun what you think about deleting this example?

Comment (kbdharun) [reply pages/common/yt-dlp.md:38]: > @kbdharun what you think about deleting this example?

I guess it's fine to delete it, as it seems to be a bit complicated.

**Review by spageektti — APPROVED** [review]

File: pages/common/yt-dlp.md
Comment (kbdharun) [inline pages/common/yt-dlp.md:16]:
```suggestion
- Download a video or playlist using the best MP4 video available (default is "bv\*+ba/b"):
```

**Review by kbdharun — APPROVED** [review]

**Pull request merged**
